### DISCUSSION TITLE
date format to follow Logseq settings

### DISCUSSION THREAD

**Discussion by hoegge** [YU000jp/logseq-plugin-confirmation-done-task#50] [Ideas]
When I comple a task the date format is shown in US style;
![image](https://github.com/YU000jp/logseq-plugin-confirmation-done-task/assets/15846310/24aaf26e-c06d-44e0-8ce0-1263551a68ed)
and not the format chosen in LogsEQ:
![image](https://github.com/YU000jp/logseq-plugin-confirmation-done-task/assets/15846310/82450806-0f3f-43af-bc40-028770b7b4d9)
Would it be possible to make the dialog follow the chosen format?

PS: The dialog is also a bit to big for the window, as you can see on the first image.

Thanks for the plug-in btw - great enhancement

**Comment by YU000jp** [comment]
That's exactly right. The display may be confusing because it differs from the actual date format. But it's difficult to change because it relies on the browser's date picker.
1. [x] Instead, we made an improvement to make the indicator icon larger. (Click the indicator icon on the right to display the picker.)
1. [x] I have adjusted the window size, so please check it at the new version.
